aibolit/ast_framework/computed_fields_catalog/chained_fields.py

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: Copyright (c) 2019-2025 Aibolit
 # SPDX-License-Identifier: MIT
 
-from typing import Union, Any, Callable
+from typing import Union, Any, Callable, List
```

Comment:
Not that it is that critical, but starting from Python 3.9 they allowed using just regular `list`, `tuple` instead of `List` and `Tuple`.
Also, a more modern way suggests avoiding `Union[X, Y]` in favor of `X | Y`.